net/spoofer/Makefile

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (C) 2018 The Regents of the University of California
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=spoofer
+PKG_VERSION:=1.3.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://www.caida.org/projects/spoofer/downloads
+PKG_HASH:=2f5076a4ddd749a26eb97f77bd02a6c69cba0f2527fd3ff2ba6d21ec7d3b4914
+
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=LICENSE
+
+HOST_BUILD_DEPENDS:=protobuf/host
+
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/package.mk
+
+$(eval $(call HostBuild))
+
+define Package/spoofer
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Measure your ISP's resistance to spoofed IP packets
+  URL:=https://spoofer.caida.org/
+  MAINTAINER:=Ken Keys <spoofer-info@caida.org>
+  DEPENDS:=+protobuf-lite +libpcap +libpthread +libopenssl
+endef
+# Note: to use standard ssl certs, add "+ca-certificates" to DEPENDS above
```

Comment:
note to who?  If this is an optional dependency, that it can use if they're available, I'd suggest at least putting it in the description below, were people have a chance of seeing it and turning on the optional extra.

Reply:
The only reason ca-certificates would be needed is if the spoofer server ever replaced its private cert and there wasn't yet an updated openwrt spoofer package containing a new matching public cert.  But I wanted to avoid depending on ca-certificates if possible because it's so large.  I can put this in the description.